### ISSUE TITLE
Error when using the CLI option

### ISSUE THREAD

**Issue by art1415926535** [zheller/flake8-quotes#49]
I try use the CLI option but get error.
```
> flake8 --quotes='"'
Usage: flake8 [options] file file ...

flake8: error: option --quotes: invalid choice: "''" (choose from '"', "'")
```
Without the option it works fine.

flake8-quotes (0.8.1)
Python 3.5
Windows 10

**Comment by twolfson** [comment]
Windows' `cmd.exe` doesn't seem to interpret neither single nor double quotes:

```
C:\Users\IEUser>echo 'hi"'
'hi"'

C:\Users\IEUser>echo "hi'"
"hi'"
```

Can you try `flake8 --quotes="`?

Arguably, choosing to use the literals was a bad call due to escaping issues =/ Maybe we should either alias them as `--quotes=double` or remove the literals entirely?

**Comment by art1415926535** [comment]
Yes, the same error.
```
flake8 --quotes="
Usage: flake8 [options] file file ...

flake8: error: option --quotes: invalid choice: '' (choose from '"', "'")
```

**Comment by twolfson** [comment]
Ah, damn. Alright, I'm going to whip up a PR with aliasing support for `--quotes=double` and `--quotes=single`. Sorry for the inconvenience and thanks for bug report and timely responses =)

**Comment by twolfson** [comment]
We have opened up the PR https://github.com/zheller/flake8-quotes/pull/50 to resolve the issue

**Comment by twolfson** [comment]
We have landed and released #50 in `0.9.0`. Thanks again for the bug report and timely responses
